.mailmap

```diff
@@ -322,7 +322,7 @@ Matthias Bastian <dev@matthias-bastian.de> <piepmatz@users.noreply.github.com>
 Maurice Hayward <mauricehayward1@gmail.com>
 Maya Lekova <apokalyptra@gmail.com> <mslekova@chromium.org>
 Michael Bernstein <michaelrbernstein@gmail.com>
-Michael Dawson <michael_dawson@ca.ibm.com> <mdawson@devrus.com>
+Michael Dawson <midawson@redhat.com> <mdawson@devrus.com> <michael_dawson@ca.ibm.com>
```

Comment:
I think, based on https://www.git-scm.com/docs/gitmailmap, that these would need to be two separate entries.

Reply:
looking elsewhere in the file, this looks like a correct assumption: https://github.com/nodejs/node/blob/d7797d085019ed1ccb51ce2e45899000effa457a/.mailmap#L286-L287

I think this is right?
```suggestion
Michael Dawson <mdawson@devrus.com> <michael_dawson@ca.ibm.com>
Michael Dawson <mdawson@devrus.com> <midawson@redhat.com>
```

Reply:
@bnb I think was trying to get them all mapped to midawson@redhat.com.  I've pushed an update to reflect that as two entries.